elements/pfe-cta/src/_shared-assets.scss

```diff
@@ -167,7 +167,6 @@ $primary: (
     arrow: (Display: none),
     BackgroundColor: pfe-var(ui-disabled),
     BorderColor: pfe-var(ui-disabled),
-    Color: pfe-var(ui-disabled--text),
-    FontSize: pfe-local(FontSize--priority)
```

Comment:
This upgrades to this: --pfe-cta--FontSize: var(--pfe-cta--FontSize--priority, calc(var(--pfe-cta--FontSize, var(--pfe-theme--font-size, 16px))/1.125)) which is self-referential and breaks IE11.